### PR TITLE
Fixing already installed apps & link to them

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Fixing already installed apps that were not showing a disabled "Installed" button as before.
<kbd>image</kbd>

Also took the opportunity to add a link to the already installed apps to their own page, in case the user wants to install it again if the app allows it.
<kbd>image</kbd>

**Environment**: Staging(main branch) / Production

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

For example, install Zapier or HubSpot, both apps without the possibility to install multiple instances. It should show in their app page that the app is already installed. And in the Installed Apps page, you can go to each app page through its title.

### CODE REVIEW

File: apps/web/components/App.tsx [hunk 1]
```diff
@@ -147,6 +147,10 @@ const Component = ({
                     />
                   )}
                 </div>
+              ) : installedAppCount > 0 ? (
+                <Button color="secondary" disabled title="App already installed">
+                  {t("installed")}
+                </Button>
```
Comment (leog) [inline apps/web/components/App.tsx:153]: Missing "Installed" button

File: apps/web/components/integrations/IntegrationListItem.tsx [hunk 2]
```diff
@@ -19,7 +21,9 @@ function IntegrationListItem(props: {
           props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />
         }
         <div className="flex-grow truncate pl-2">
-          <ListItemTitle component="h3">{props.title}</ListItemTitle>
+          <ListItemTitle component="h3">
+            <Link href={"/apps/" + props.slug}>{props.title}</Link>
```
Comment (leog) [inline apps/web/components/integrations/IntegrationListItem.tsx:25]: Link to app page from Installed Apps

Comment (PeerRich) [reply apps/web/components/integrations/IntegrationListItem.tsx:25]: great idea

**Review by PeerRich — APPROVED** [review]
good changes!

**Pull request merged**